Cargo.toml

```diff
@@ -50,7 +50,8 @@ all = ["lib", "cli", "s3"]
 lib = ["remote", "gz", "bz", "lz", "json"]
 s3= ["rust-s3", "dotenvy"]
 
-remote=["reqwest", "suppaftp"]
+remote=["reqwest/default-tls", "suppaftp"]
+rustls = ["reqwest?/rustls-tls"]
```

Comment:
I opted to use the `?` syntax here as described in the [Cargo Features documentation](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features):

> The `"package-name/feature-name"` syntax will also enable `package-name` if it is an optional dependency. Often this is not what you want. You can add a `?` as in `"package-name?/feature-name"` which will only enable the given feature if something else enables the optional dependency.

My thinking here was that if there end up being other dependencies that have `rustls` feature flags, we could add them to this flag without having it enable `reqwest` if someone opted not to have the `remote` flag enabled. Curious on thoughts here.

Reply:
Looking more closely at suppaftp, it looks like they also have a rustls feature flag that can be set. I've went ahead and updated the rustls to include use that flag in 92e4f9a8b666b3c25d61b8d784a6b98bc5f5a9a6.